### DISCUSSION TITLE
Wharf and ship upkeep

### DISCUSSION THREAD

**Discussion by q6276270** [Keriew/augustus#605] [Ideas]
In GamerZakh's [latest](https://www.youtube.com/watch?v=gwMwRgaTHVk) Thera playthrough it slightly annoyed me how you can easily delete the wharf once you have all the ships you need. So I propose a mechanic for preventing this:

(1) Make fishing vessels take minor damage over time requiring occasional minor repairs in their quays. The repairs may take a few seconds of inactivity at the next visit to the quay (or similar).

(2) Make fishing vessels take major damage over time which can only be solved by sending the ship to a functioning wharf. To be perfectly clear: The quay would send the ship to the wharf automatically if a functioning wharf exists. If the damage is not repaired the vessel may either just stay in the quay inactive until resolved or even sink (don't know if ship sinking exists within Ceasar/Julius/Augustus yet).

I imagine both risks could be modeled similarly to building risks such as fire but obviously there are also other models.

**Comment by Riww** [comment]
Fishing ships can be destroyed after some time, then the wharf would build a new one automatically. Enemy archers may also target them.  I'm against the need of the user to micromanage, there is enough work as it is.

**Reply by q6276270** [reply]
I am not suggesting more work. Quite the opposite. To have a functioning wharf around would reduce the amount of necessary micromanagement... The way it works now is you can just remove the wharf once you have equipped all the quays with ships. It doesn't seem to me that this is the way the system is supposed to work.
